ui/tests/acceptance/sync/secrets/overview-test.js

```diff
@@ -162,8 +162,8 @@ module('Acceptance | sync | overview', function (hooks) {
       this.server.post('/sys/activation-flags/secrets-sync/activate', (_, req) => {
         assert.strictEqual(
           req.requestHeaders['X-Vault-Namespace'],
-          'root',
-          'Request is made to root namespace'
+          'admin/foo',
```

Comment:
👀  how was this passing before? did we miss it in an earlier pr?

Reply:
there's something off about this test. I'll add a note to myself to investigate tmrw. But looking at the test it makes sense you'd be in the admin/foo namespace so I have no idea how it passed before. I suspect on your PR you'll hit flakyness with it.